paper.bib

```diff
@@ -133,81 +125,40 @@ @article{ramachandran2011mayavi
   doi={10.1109/mcse.2011.35}
 }
 
-@article{garyfallidis2014dipy,
-  title={Dipy, a library for the analysis of diffusion MRI data},
-  author={Garyfallidis, Eleftherios and Brett, Matthew and Amirbekian, Bagrat and Rokem, Ariel and Van Der Walt, Stefan and Descoteaux, Maxime and Nimmo-Smith, Ian},
-  journal={Frontiers in neuroinformatics},
-  volume={8},
-  pages={8},
-  year={2014},
-  publisher={Frontiers},
-  doi={10.3389/fninf.2014.00008}
-}
-
-
-
-@article{Pearson:2017,
-  	url = {http://adsabs.harvard.edu/abs/2017arXiv170304627P},
-  	Archiveprefix = {arXiv},
-  	Author = {{Pearson}, S. and {Price-Whelan}, A.~M. and {Johnston}, K.~V.},
-  	Eprint = {1703.04627},
-  	Journal = {ArXiv e-prints},
-  	Keywords = {Astrophysics - Astrophysics of Galaxies},
-  	Month = mar,
-  	Title = {{Gaps in Globular Cluster Streams: Pal 5 and the Galactic Bar}},
-  	Year = 2017
-}
-
-@book{Binney:2008,
-  	url = {http://adsabs.harvard.edu/abs/2008gady.book.....B},
-  	Author = {{Binney}, J. and {Tremaine}, S.},
-  	Booktitle = {Galactic Dynamics: Second Edition, by James Binney and Scott Tremaine.~ISBN 978-0-691-13026-2 (HB).~Published by Princeton University Press, Princeton, NJ USA, 2008.},
-  	Publisher = {Princeton University Press},
-  	Title = {{Galactic Dynamics: Second Edition}},
-  	Year = 2008
+@inproceedings{rougier2015glumpy,
+  title={Glumpy},
+  author={Rougier, Nicolas P},
+  booktitle={EuroScipy},
+  year={2015}
 }
 
-@article{gaia,
-    author = {{Gaia Collaboration}},
-    title = "{The Gaia mission}",
-    journal = {Astronomy and Astrophysics},
-    archivePrefix = "arXiv",
-    eprint = {1609.04153},
-    primaryClass = "astro-ph.IM",
-    keywords = {space vehicles: instruments, Galaxy: structure, astrometry, parallaxes, proper motions, telescopes},
-    year = 2016,
-    month = nov,
-    volume = 595,
-    doi = {10.1051/0004-6361/201629272},
-    url = {http://adsabs.harvard.edu/abs/2016A%26A...595A...1G},
+@book{schroeder2004visualization,
+  title={The visualization toolkit: an object-oriented approach to 3D graphics},
+  author={Schroeder, Will J and Lorensen, Bill and Martin, Ken},
+  year={2004},
+  publisher={Kitware}
 }
```

Comment:
can you replace by:
```tex
@inproceedings{schroeder1996visualization,
  author={Schroeder, W.J. and Martin, K.M. and Lorensen, W.E.},
  booktitle={Proceedings of Seventh Annual IEEE Visualization '96},
  title={The design and implementation of an object-oriented toolkit for 3D graphics and visualization},
  year={1996},
  volume={},
  number={},
  pages={93-100},
  doi={10.1109/VISUAL.1996.567752}
}
```

Reply:
This is a new reference that cites the 1996 paper.

Reply:
This is the same but older (the older one is the paper, and the other one is the book if I am not wrong). I personally prefer this reference.

Reply:
Added.